autotest.pm

```diff
@@ -45,7 +45,8 @@ loadtest is called.
 =cut
 
 sub find_script ($script) {
-    if (defined(my $wheel = (glob Cwd::getcwd . "/*/tests/$script"))) {
+    my $wheels_dir = $bmwqemu::vars{WHEELS_DIR} // Cwd::getcwd . "/*/tests/";
+    if (defined(my $wheel = (glob $bmwqemu::vars{WHEELS_DIR} . "/*/tests/$script"))) {
```

Comment:
```suggestion
    my $wheels_dir = $bmwqemu::vars{WHEELS_DIR} // Cwd::getcwd;
    if (defined(my $wheel = (glob $wheels_dir . "/*/tests/$script"))) {
```

Reply:
Thanks! I didn't realize I hadn't replaced that correctly

Reply:
I intentionally left out the `. "/*/tests/"` in the first line for a reason. See my other comment.